docs/UserGuide.md

```diff
@@ -1087,24 +1137,33 @@ That is not a valid command.
 
-**Q: What is considered a duplicate in Connectify?**
+**Q2: What is considered a duplicate in Connectify?**
 
 A: In Connectify, companies and people are considered duplicates if they share the same name as an existing company or person. Note that this condition holds for names that are the same but which have the same case.
 
+**Q3: Why are my fields being cut off in the UI?**
 
-**Q: Why are my names being cut off in the UI?**
+A: Do note that Connectify's UI does not fully support long fields and will cut off fields that are too long. However, the supported length is likely sufficient for most fields. 
 
-A: Do note that Connectify's UI does not fully support long names and will cut off names that are too long. However, the supported length is likely sufficient for most names. 
-
-**Q: What sort of inputs are not allowed in Connectify?**
+**Q4: What sort of inputs are not allowed in Connectify?**
 
 A: Connectify does not impose strict input validation on the fields of companies and people, other than basic ones such as Names needing to be alphanumeric, not blank, and case-sensitive unique, while phone numbers must be numeric. Stricter input validation will be implemented in future versions of Connectify.
 
+**Q5: What if one of my contacts belongs to multiple companies?**
+
+A: Connectify does not support the same person being associated with multiple companies at our current version. This is to avoid confusion when sharing contacts with others. However, we are looking to implement this feature in future versions of Connectify.
+
+**Q6: Should a person of higher priority be assigned a higher or lower priority number?**
+
+A: A person of a higher priority should be assigned a higher priority number. This is because Connectify's `rank` command ranks people by decreasing numerical values of priority. Hence, a person with priority 10 will be ranked higher (closer to the top of the list) than a person with priority 1. The numerical range for the priority field is 1 to 10.
+
+**Q7: What is the numerical range for the priority field?**
 
+A: Connectify does not restrict the numerical range for the priority field. However, the numerical range for the priority field is 1 to 10. This is because Connectify's `rank` command ranks people by decreasing numerical values of priority. This is so that a person who has ranked contacts of the highest priority at point in time wouldn't need to re-rank their contacts if they wish to add a new contact of an even higher priority in the future. However, the numerical value for priority should not be a negative number.
```

Comment:
these FAQs are also very helpful for users to better understand the rationale behind our commands.